plexapi/video.py

```diff
@@ -622,3 +643,31 @@ def show(self):
     def _defaultSyncTitle(self):
         """ Returns str, default title for a new syncItem. """
         return '%s - %s - (%s) %s' % (self.grandparentTitle, self.parentTitle, self.seasonEpisode, self.title)
+
+    def subtitleStreams(self):
+        """ Returns a list of :class:`~plexapi.media.SubtitleStream` objects for all MediaParts. """
+        streams = []
+        for elem in self.media:
+            for part in elem.parts:
+                streams += part.subtitleStreams()
+        return streams
+
+    def uploadSubtitles(self, filepath):
+        """ Upload Subtitle file for video. """
+        url = '%s/subtitles' % self.key
+        filename = os.path.basename(filepath)
+        subFormat = os.path.splitext(filepath)[1][1:]
+        with open(filepath, 'rb') as subfile:
+            params = {'title': filename,
+                      'format': subFormat
+                      }
+            headers = {'Accept': 'text/plain, */*'}
+            self._server.query(url, self._server._session.post, data=subfile, params=params, headers=headers)
+        self.reload()
+
+    def removeSubtitles(self, streamID=int, streamTitle=str):
```

Comment:
We dont use any typing so remove streamID=int, streamTitle=str abd just use None, you you want it to be a empty string for some reason use ''

Reply:
Also, does this delete the subtitle on the server or just disables it for this item?

Reply:
It depends on where the subtitles file is located. If it's inside the video's folder then it will be deleted. If outside of the video's folder then Plex will not remove it. I should probably add that in the docstring.

Reply:
Can you fix the doc string?

Reply:
It's been [updated](https://github.com/blacktwin/python-plexapi/blob/patch-8/plexapi/video.py#L116-L120).